app/views/students/index.html

```diff
@@ -42,22 +42,22 @@ <h1 translate='i18nStudentsPanelSection' class='page-header'></h1>
       <table class='table table-hover'>
         <thead>
           <tr>
-            <th translate='i18Docket'></th>
+            <th translate='i18nDocket'></th>
             <th translate='i18nFirstName'></th>
             <th translate='i18nLastName'></th>
-            <th translate='i18Actions'></th>
+            <th translate='i18nActions'></th>
           </tr>
         </thead>
         <tbody>
           <tr ng-hide='controller.students.length > 0' class='bg-warning'>
-            <td colspan='4' class='text-danger text-center' translate='i18NoStudentsFound'></td>
+            <td colspan='4' class='text-danger text-center' translate='i18nNoStudentsFound'></td>
           </tr>
           <tr ng-repeat='student in controller.students'>
             <td>{{ student.docket }}</td>
             <td>{{ student.firstName }}</td>
             <td>{{ student.lastName }}</td>
             <td class='actions-container'>
-              <a class='btn btn-default fullWidthButton' role='button'>
+              <a class='btn btn-default full-width-button' role='button'>
```

Comment:
Now, reading this, I think that this class (`full-width-button`) is not necessary, as we are using flex to fill the space depending on the amount of action buttons. I'll check it later.